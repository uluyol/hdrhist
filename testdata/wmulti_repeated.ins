0
1
12
11
12
---
1
2
1
1
---
2
3
1
1
---
3
4
---
