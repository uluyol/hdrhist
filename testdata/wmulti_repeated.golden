0.000,0.001,0.000,HISTFAAAAB542pNpmSzMwMDAzAABMJoRSjPZf4AwRJlYAEJSApo=
0.001,0.001,0.000,HISTFAAAAB142pNpmSzMwMDAxAABzFCaEUoz2X+AslgAP20Cgg==
0.002,0.001,0.000,HISTFAAAAB142pNpmSzMwMDAxAABzFCaEUoz2X+AslgAP20Cgg==
0.003,0.001,0.000,HISTFAAAABx42pNpmSzMwMDAyAABzFAaxmey/wBlAQA8yQJ9
